### PR TITLE
docs(pill): update docs

### PR DESCRIPTION
**Pull request by thebesson**

### CODE REVIEW

File: packages/components/pill/README.mdx [hunk 1]
```diff
@@ -8,58 +8,41 @@ storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-pill--b
 typescript: ./src/Pill.tsx
 ---
 
-import { InfoCircleIcon } from '@contentful/f36-icons';
-
 Pills represent a set of one or multiple objects which can be entered or changed by the user. By default, pills can be removed by the user. When necessary, the order of pills can be changed by the user using drag and drop. They are used to represent complex entities in small blocks such as contact details. Contentful Pills are are used to render transition locales and tag elements.
```
Comment (bgutsol) [inline packages/components/pill/README.mdx:11]:
```suggestion
Pills represent a set of one or multiple objects which can be entered or changed by the user. By default, pills can be removed by the user. When necessary, the order of pills can be changed by the user using drag and drop. They are used to represent complex entities in small blocks such as contact details. Contentful Pills are used to render transition locales and tag elements.
```

Could you rephrase this description a bit? For example, first three sentences don't really make sense to me

File: packages/components/pill/README.mdx [hunk 2]
```diff
@@ -8,58 +8,41 @@ storybook: 'https://v4-f36-storybook.netlify.app/?path=/story/components-pill--b
...
 
-## Code examples
+## Examples
 
-Example of usage with draggable and close option
+### Using with draggable and close option
 
-```jsx
-// import { Pill } from '@contentful/f36-pill';
+```jsx file=./examples/DraggableAndClosePillExample.tsx
```
Comment (bgutsol) [inline packages/components/pill/README.mdx:25]: Don't know what is a better way to improve those two examples, but they feel kind of useless. Especially the second one with custom dragHandleComponent. WDYT, can we improve them somehow?

**Review by bgutsol — COMMENTED** [review]
Couple of comments

File: packages/components/pill/README.mdx (hunk 2 above)
Comment (gui-santos) [reply packages/components/pill/README.mdx:25]: I agree with you, it seems that the second one could be removed

Maybe there are some use cases in webapp

Comment (thebesson) [reply packages/components/pill/README.mdx:25]: I decided to keep those basic examples, just in case someone would need to just copy a basic one, and I added an example with react-sortable-hoc. let me know what do you think :)

File: packages/components/pill/examples/IntegrationPillExample.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+      useDragHandle
+      axis="xy"
+      distance={10}
+      onSortEnd={({ oldIndex, newIndex }) => {
+        swapItems({ oldIndex, newIndex });
+      }}
+    >
+      {items.map((item, index) => (
+        <SortablePill
+          key={item + index}
```
Comment (bgutsol) [inline packages/components/pill/examples/IntegrationPillExample.tsx:72]: why do you need an index here? It's kind of good practise not to use the index as keys for items that can change their order.

**Review by bgutsol — APPROVED** [review]
All good 👍
Just one minor comment

File: packages/components/pill/examples/IntegrationPillExample.tsx (hunk 3 above)
Comment (thebesson) [reply packages/components/pill/examples/IntegrationPillExample.tsx:72]: We want it to be re-rendered, right?

Comment (bgutsol) [reply packages/components/pill/examples/IntegrationPillExample.tsx:72]: Yes, but in our case item is unique, so we can use only it for the `key` prop, right?
